src/__test__/components/FavoriteActivities.test.tsx

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-undef */
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import FavoriteActivities from '../../components/FavoriteActivities/index';
+import '@testing-library/jest-dom';
+
+// empty section should be tested in the future
```

Comment:
Question: if we delete all cards, doesn't the empty section appears? If so, I think we can add this test now.

Reply:
It will be complex to delete them all just to test this, and as this logic will change in the future with the integration with the backend